### PR TITLE
#312 - handle empty files upload

### PR DESCRIPTION
**Pull request by yzerk**

### CODE REVIEW

File: src/main/resources/messages/messages.properties [hunk 1]
```diff
@@ -260,6 +260,7 @@ error.cast_param_list_type='%s' list contains value(s) of type '%s' instead of '
 error.dest_and_pattern_in_source=The 'dest' parameter only works for single files specified in the 'source' parameter
 error.dest_and_preserve_hierarchy=The 'dest' parameter only works for single files with the specified 'preserve_hierarchy': true option
 error.upload_to_storage=Failed to upload the '%s' file to the storage. Please contact our support team for help
+error.empty_file_upload=Wrong parameters: %s
```
Comment (andrii-bodnar) [inline src/main/resources/messages/messages.properties]: unused string

File: src/main/java/com/crowdin/cli/commands/actions/UploadSourcesAction.java [hunk 2]
```diff
@@ -259,6 +260,10 @@ public void act(Outputter out, PropertiesWithFiles pb, ProjectClient client) {
 
                                 try (InputStream fileStream = new FileInputStream(sourceFile)) {
                                     request.setStorageId(client.uploadStorage(source.substring(source.lastIndexOf(Utils.PATH_SEPARATOR) + 1), fileStream));
+                                } catch (EmptyFileException e){
+                                    errorsPresented.set(false);
+                                    out.println(WARNING.withIcon(String.format(RESOURCE_BUNDLE.getString("message.uploading_file_skipped"), fileFullPath)));
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/cli/commands/actions/UploadSourcesAction.java]: `SKIPPED.withIcon`

File: src/main/java/com/crowdin/cli/client/EmptyFileException.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.crowdin.cli.client;
+
+
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/cli/client/EmptyFileException.java]: minor thing - extra line

File: src/main/resources/messages/messages.properties (hunk 1 above)
Comment (yzerk) [reply src/main/resources/messages/messages.properties]: done

File: src/main/java/com/crowdin/cli/client/EmptyFileException.java (hunk 3 above)
Comment (yzerk) [reply src/main/java/com/crowdin/cli/client/EmptyFileException.java]: done

File: src/main/java/com/crowdin/cli/commands/actions/UploadSourcesAction.java (hunk 2 above)
Comment (yzerk) [reply src/main/java/com/crowdin/cli/commands/actions/UploadSourcesAction.java]: done

**Review by andrii-bodnar — APPROVED** [review]

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**